phpBB/includes/acp/acp_attachments.php

```diff
@@ -226,7 +225,7 @@ function main($id, $mode)
 				}
 
 				// We go through the display_vars to make sure no one is trying to set variables he/she is not allowed to...
-				foreach ($display_vars['vars'] as $config_name => $null)
+				foreach (array_keys($display_vars['vars']) as $config_name)
```

Comment:
Giving up any performance or extra memory so that my IDE doesn't color a variable grey doesn't seem like a sound tradeoff. Also, this exact pattern is in the PHP docs: https://www.php.net/manual/en/function.array-key-first.php#refsect1-function.array-key-first-notes